packages/camera/camera_windows/windows/camera.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "camera.h"
+
+namespace camera_windows {
+using flutter::EncodableList;
+using flutter::EncodableMap;
+using flutter::EncodableValue;
+
+namespace {
+// Camera channel events
+const char kCameraMethodChannelBaseName[] = "flutter.io/cameraPlugin/camera";
+const char kVideoRecordedEvent[] = "video_recorded";
+const char kErrorEvent[] = "error";
+
+// Helper function for creating messaging channel for camera
```

Comment:
Why are you constantly creating new method channels instead of keeping them around?